### ISSUE TITLE
powerpack-cli fails to install

### ISSUE THREAD

**Issue by jivimberg** [rossmacarthur/powerpack#4]
Running `cargo install powerpack-cli` fails with the following errors:

```
   Compiling powerpack-cli v0.2.1
error[E0432]: unresolved import `clap::Clap`
  --> /Users/jvimberg/.cargo/registry/src/github.com-1ecc6299db9ec823/powerpack-cli-0.2.1/src/main.rs:12:25
   |
12 | use clap::{AppSettings, Clap};
   |                         ^^^^ no `Clap` in the root

error: cannot determine resolution for the derive macro `Clap`
   --> /Users/jvimberg/.cargo/registry/src/github.com-1ecc6299db9ec823/powerpack-cli-0.2.1/src/main.rs:169:17
    |
169 | #[derive(Debug, Clap)]
    |                 ^^^^
    |
    = note: import resolution is stuck, try simplifying macro imports

error: cannot find attribute `clap` in this scope
   --> /Users/jvimberg/.cargo/registry/src/github.com-1ecc6299db9ec823/powerpack-cli-0.2.1/src/main.rs:174:11
    |
174 |         #[clap(long)]
    |           ^^^^
    |
    = note: `clap` is in scope, but it is a crate, not an attribute

error: cannot find attribute `clap` in this scope
   --> /Users/jvimberg/.cargo/registry/src/github.com-1ecc6299db9ec823/powerpack-cli-0.2.1/src/main.rs:180:11
    |
180 |         #[clap(long)]
    |           ^^^^
    |
    = note: `clap` is in scope, but it is a crate, not an attribute

error: cannot find attribute `clap` in this scope
   --> /Users/jvimberg/.cargo/registry/src/github.com-1ecc6299db9ec823/powerpack-cli-0.2.1/src/main.rs:185:11
    |
185 |         #[clap(long)]
    |           ^^^^
    |
    = note: `clap` is in scope, but it is a crate, not an attribute

error: cannot determine resolution for the derive macro `Clap`
   --> /Users/jvimberg/.cargo/registry/src/github.com-1ecc6299db9ec823/powerpack-cli-0.2.1/src/main.rs:194:17
    |
194 | #[derive(Debug, Clap)]
    |                 ^^^^
    |
    = note: import resolution is stuck, try simplifying macro imports

error: cannot find attribute `clap` in this scope
   --> /Users/jvimberg/.cargo/registry/src/github.com-1ecc6299db9ec823/powerpack-cli-0.2.1/src/main.rs:195:3
    |
195 | #[clap(
    |   ^^^^
    |
    = note: `clap` is in scope, but it is a crate, not an attribute

error: cannot find attribute `clap` in this scope
   --> /Users/jvimberg/.cargo/registry/src/github.com-1ecc6299db9ec823/powerpack-cli-0.2.1/src/main.rs:206:7
    |
206 |     #[clap(subcommand)]
    |       ^^^^
    |
    = note: `clap` is in scope, but it is a crate, not an attribute

error[E0599]: no function or associated item named `parse` found for struct `Opt` in the current scope
   --> /Users/jvimberg/.cargo/registry/src/github.com-1ecc6299db9ec823/powerpack-cli-0.2.1/src/main.rs:211:32
    |
205 | struct Opt {
    | ---------- function or associated item `parse` not found for this
...
211 |     let Opt { command } = Opt::parse();
    |                                ^^^^^ function or associated item not found in `Opt`
    |
    = help: items from traits can only be used if the trait is implemented and in scope
    = note: the following traits define an item `parse`, perhaps you need to implement one of them:
            candidate #1: `combine::parser::Parser`
            candidate #2: `combine::parser::ParseMode`
            candidate #3: `Parser`
            candidate #4: `time::parsing::parsable::sealed::Sealed`

Some errors have detailed explanations: E0432, E0599.
For more information about an error, try `rustc --explain E0432`.
error: failed to compile `powerpack-cli v0.2.1`, intermediate artifacts can be found at `/var/folders/6c/wbbn4x0s393716pfk79xwttc0000gn/T/cargo-installK4t9kX`

Caused by:
  could not compile `powerpack-cli` due to 9 previous errors

```

**Comment by jivimberg** [comment]
Works if I run `cargo install powerpack-cli --locked`, so I'm guessing a dependency update broke compilation

**Comment by rossmacarthur** [comment]
Yeah I was about to suggest that. `powerpack-cli` is using a beta version of `clap` but I should have pinned it. It looks like `clap` has finally released v3 though, so the dep just needs to be updated to that.

**Comment by rossmacarthur** [comment]
Resolved in 35f2a7c9833ce85df420b6b9980f674679173163
